### PR TITLE
Subscriptions: Buildurl fix

### PR DESCRIPTION
**Pull request by prateekbh**
- uses build url for authorization

### CODE REVIEW

File: extensions/amp-subscriptions/0.1/local-subscription-platform.js [hunk 1]
```diff
@@ -183,16 +183,15 @@ export class LocalSubscriptionPlatform {
...
-        .fetchJson(this.authorizationUrl_, {
-          credentials: 'include',
-        })
-        .then(res => res.json())
-        .then(resJson => {
-          const entitlement = Entitlement.parseFromJson(resJson);
-          this.entitlement_ = entitlement;
-          return entitlement;
-        });
+    return this.urlBuilder_.buildUrl(this.authorizationUrl_, false)
```
Comment (dvoytenko) [inline extensions/amp-subscriptions/0.1/local-subscription-platform.js]: Add `/* useAuthData */` before `false`.

File: extensions/amp-subscriptions/0.1/local-subscription-platform.js [hunk 2]
```diff
@@ -183,16 +183,15 @@ export class LocalSubscriptionPlatform {
...
+        .then(fetchUrl =>
+          this.xhr_.fetchJson(fetchUrl, {credentials: 'include'})
```
Comment (dvoytenko) [inline extensions/amp-subscriptions/0.1/local-subscription-platform.js]: Looks good. Let's add a test.

File: extensions/amp-subscriptions/0.1/local-subscription-platform.js (hunk 1 above)
Comment (prateekbh) [reply extensions/amp-subscriptions/0.1/local-subscription-platform.js]: Done

File: extensions/amp-subscriptions/0.1/local-subscription-platform.js (hunk 2 above)
Comment (prateekbh) [reply extensions/amp-subscriptions/0.1/local-subscription-platform.js]: Done

**Pull request merged**